demisto_sdk/tests/constants_test.py

```diff
@@ -49,7 +49,7 @@
 INVALID_MULTI_LINE_1_CHANGELOG_PATH = f"{GIT_ROOT}/demisto_sdk/tests/test_files/invalid-multi-line_1_CHANGELOG.md"
 INVALID_MULTI_LINE_2_CHANGELOG_PATH = f"{GIT_ROOT}/demisto_sdk/tests/test_files/invalid-multi-line_2_CHANGELOG.md"
 LAYOUT_TARGET = "./Layouts/layout-mock.json"
-REPUTATION_TARGET = "./Misc/reputations.json"
+INDICATOR_TYPE_TARGET = "IndicatorTypes/reputations.json"
```

Comment:
I don't think that it it is good that our example of `INDICATOR_TYPE_TARGET` is the deprecated exception `reputations.json`